frontend/src/components/filters/FiltersItem.js

```diff
@@ -508,8 +509,18 @@ class FiltersItem extends PureComponent {
                 {filter.parameters &&
                   filter.parameters.map((item, index) => {
                     const { widgetType } = item;
+                    let autoFocus = false;
                     item.field = item.parameterName;
 
+                    // don't focus date/time related fields
+                    if (
+                      !autoFocusedField &&
+                      !Object.keys(DATE_FIELD_FORMATS).includes(widgetType)
```

Comment:
what's the reason to avoid giving focus to date/time components?

Reply:
Because it's a custom widget, there's no input field we can immediately focus. If I comment this out, field won't be focus until user clicks `[tab]` again

![0DOSbjw6Eg](https://user-images.githubusercontent.com/513460/147011823-d6d29278-2c46-434e-ac32-a1d0c61fa729.gif)

Reply:
@siemiatj ok, but then why not extracting this to some `isFocusableWidgetType(widgetType)` function?
There u could add a comment that dates are not focusable because those components don't have an input field (yet)...